deploy/manifests/nginx-gateway.yaml

```diff
@@ -86,7 +86,7 @@ spec:
         - name: nginx-config
           mountPath: /etc/nginx
       containers:
-      - image: nginx-kubernetes-gateway:edge
+      - image: ghcr.io/nginxinc/nginx-kubernetes-gateway:edge
         imagePullPolicy: IfNotPresent
```

Comment:
for the edge tag, better to have `imagePullPolicy: Always` because the edge tag changes on every commit and the `IfNotPresent` tells the node to cache the `edge` tag forever.